File: data/src/main/java/com/balv/imdb/data/model/SearchData.java
package com.balv.imdb.data.model;

import com.google.gson.annotations.SerializedName;

import java.util.ArrayList;

public class SearchData {
    @SerializedName("Search")
    public ArrayList<MovieNetworkObject> list;
    public String totalResults;
    @SerializedName("Response")
    public String response;
}

File: data/src/main/java/com/balv/imdb/data/model/SearchData.kt
package com.balv.imdb.data.model

import com.google.gson.annotations.SerializedName

data class SearchData(
    @SerializedName("Search")
    val list: ArrayList<MovieNetworkObject>? = null,
    val totalResults: String? = null,
    @SerializedName("Response")
    val response: String? = null
)

